flocker_bb/builders/flocker.py

```diff
@@ -128,10 +127,8 @@ def _flockerCoverage():
             ),
         DirectoryUpload(
             b"change-coverage-annotations",
-            Interpolate(b"private_html/%s/%s/change" % (branch, revision)),
-            url=Interpolate(
-                b"/results/%s/%s/change" % (
-                    branch, revision)),
+            resultPath('change-coverage'),
+            url=resultURL('change-coverage'),
```

Comment:
It should put a `!` all new lines that aren't covered. It doesn't handle existing lines that become uncovered, though.